packages/runtime/test/transport/identityDeletionProcess.test.ts

```diff
@@ -0,0 +1,152 @@
+        });
+
+        test("should return an error trying to initiate an Identity deletion process if there already is one", async function () {
+            await transportService.identityDeletionProcesses.initiateIdentityDeletionProcess();
+            const result = await transportService.identityDeletionProcesses.initiateIdentityDeletionProcess();
+            expect(result).toBeAnError(
+                "There is already an active identity deletion process. You cannot start another, as there may only be one active identity deletion process per identity.",
+                "error.runtime.identity.activeIdentityDeletionProcessAlreadyExists"
+            );
+        });
+    });
+
+    describe(GetIdentityDeletionProcessUseCase.name, () => {
+        test("should get the active Identity deletion process without specifying an ID", async function () {
+            const initiateResult = await transportService.identityDeletionProcesses.initiateIdentityDeletionProcess();
+            const initiatedIdentityDeletionProcess = initiateResult.value;
+
+            const result = await transportService.identityDeletionProcesses.getIdentityDeletionProcess({});
+            expect(result).toBeSuccessful();
+
+            const identityDeletionProcess = result.value;
+            expect(identityDeletionProcess).toStrictEqual(initiatedIdentityDeletionProcess);
+        });
+
+        test("should get an Identity deletion process specifying an ID", async function () {
+            const initiateResult = await transportService.identityDeletionProcesses.initiateIdentityDeletionProcess();
+            const initiatedIdentityDeletionProcess = initiateResult.value;
+
+            const result = await transportService.identityDeletionProcesses.getIdentityDeletionProcess({ id: initiatedIdentityDeletionProcess.id });
+            expect(result).toBeSuccessful();
+
+            const identityDeletionProcess = result.value;
+            expect(identityDeletionProcess).toStrictEqual(initiatedIdentityDeletionProcess);
+        });
+
+        test("should return an error trying to get an Identity deletion process without specifying an ID if there is none active", async function () {
+            const result = await transportService.identityDeletionProcesses.getIdentityDeletionProcess({});
+            expect(result).toBeAnError("No active identity deletion process found.", "error.runtime.identity.noActiveIdentityDeletionProcess");
+        });
+
+        test("should return an error trying to get an Identity deletion process specifying an unknown ID", async function () {
+            const unknownId = (await CoreId.generate("IDP")).toString();
+            const result = await transportService.identityDeletionProcesses.getIdentityDeletionProcess({ id: unknownId });
+            expect(result).toBeAnError("IdentityDeletionProcess not found. Make sure the ID exists and the record is not expired.", "error.runtime.recordNotFound");
```

Comment:
I would prefer the spelling ‘IdentityDeletionProcess’ used here to be used in all other error messages too instead of ‘identity deletion process’. However, if you decide to keep ‘identity deletion process’ in the other error messages for now, I would at least consistently use ‘Identity deletion process’ instead of ‘identity deletion process’ (as you already do in the tests' names).

Reply:
we changed it to IdentityDeletionProcess everywhere it is better :)